quickwit-core/src/indexing/split.rs

```diff
@@ -243,6 +250,29 @@ impl Split {
         let manifest = put_split_files_to_storage(&*self.storage, self).await?;
         Ok(manifest)
     }
+
```

Comment:
I think we have this in the indexer and the packager right now. Am I missing something?

Reply:
Ah I see, this is because you wanted to populate tags in the current pipeline and the new one!

Reply:
Yes, currently there are two indexing pipelines. I am aware this one in split will be replaced so I did not bother duplicating just thing are in to write tests for it.